staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -69,6 +69,9 @@ func (a *mutatingDispatcher) Dispatch(ctx context.Context, attr *generic.Version
 			}
 			glog.Warningf("Failed calling webhook, failing closed %v: %v", hook.Name, err)
 		}
+		if statusError, ok := err.(*apierrors.StatusError); ok {
+			return statusError
+		}
 		return apierrors.NewInternalError(err)
```

Comment:
Do you have any insight about this line? What do we transform into an internal error at all?

Reply:
I would expect that, like for the validating dispatcher.

Do we have any e2e / integration coverage for the error message?

Reply:
seems mutatingDispatcher.callAttrMutatingHook() only return two kinds of err: 

1. StatusError

1. ErrCallingWebhook

So I think no need to wrap the err again `apierrors.NewInternalError(err)`, directly return err is ok.

Reply:
> I would expect that, like for the validating dispatcher.

Done. 

> Do we have any e2e / integration coverage for the error message?

The e2e test check the error message, but doesn't check the http status code.
https://github.com/kubernetes/kubernetes/blob/63c90bb47ec99247cbf29542edfc443ffda7298b/test/e2e/apimachinery/webhook.go#L624

I add status code code check in the unit test.